fpna/DataModel/DataStructure.ipynb

```diff
@@ -0,0 +1,253 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "bpCHQp0px0SSZnf-SYxkzQ",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Basics"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"../Constants/Ifrs17CalculationEngine\"",
+                "\n#!import \"../Constants/Enums\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "using Systemorph.Vertex.Grid.Model;",
+                "\nusing Systemorph.Vertex.Pivot.Builder.Interfaces;",
+                "\nusing Systemorph.Vertex.Hierarchies;"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Dimensions"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## AmountType"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public record RatioAmountType : AmountType {}",
+                "\npublic record PatternAmountType : AmountType {}",
+                "\npublic record AssetAmountType : AmountType {}",
+                "\npublic record SolvencyBsAmountType : AmountType {}",
+                "\npublic record IfrsBsAmountType : AmountType {}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Cashflow Definition"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public record CashflowDefinition : IKeyed",
```

Comment:
Maybe just Cashflow? It is shorter and I don't see why we need definition in the type.

Reply:
renamed